$NetBSD: patch-ah,v 1.1 2011/06/27 19:46:50 gls Exp $
gnutls uses "nettle" instead of libgcrypt per default now. So a client
 shouldn't try to deal with libgcrypt unless it uses it explicitely.

--- configure.orig	2011-04-09 08:51:04.000000000 +0000
+++ configure
@@ -16584,13 +16584,13 @@ _ACEOF
 
   LIBS="-lgpg-error $LIBS"
 
-else
-  ac_cv_enable_gnutls=no
+#else
+#  ac_cv_enable_gnutls=no
 fi
 
 
-else
-  ac_cv_enable_gnutls=no
+#else
+#  ac_cv_enable_gnutls=no
 fi
 
 
@@ -16642,13 +16642,13 @@ _ACEOF
 
   LIBS="-lgcrypt $LIBS"
 
-else
-  ac_cv_enable_gnutls=no
+#else
+#  ac_cv_enable_gnutls=no
 fi
 
 
-else
-  ac_cv_enable_gnutls=no
+#else
+#  ac_cv_enable_gnutls=no
 fi
 
 
@@ -16657,7 +16657,7 @@ fi
   	if test x"$platform_win32" = "xyes"; then
 		GNUTLS_LIBS="-lgnutls -lgcrypt ${GCRYPT_LIBS}"
 	else
-		GNUTLS_LIBS="-lgnutls -lgcrypt -lz ${GCRYPT_LIBS}"
+		GNUTLS_LIBS="-lgnutls"
 	fi
   fi
 else

